matcher/service/openreview_interface.py

```diff
@@ -116,6 +89,36 @@ def __init__(self, client, config_note_id, logger=logging.getLogger(__name__), c
                 self.set_status('Error')
                 raise error_handle
 
+    def get_all_edges(self, client, edge_invitation_id):
+        '''Helper function for retrieving and parsing all edges in bulk'''
+
+        all_edges = []
+        all_papers = {p:i for i,p in enumerate(self.papers)}
```

Comment:
`all_papers` and `all_reviewers` can be properties of the class, no need to compute this every time you call `get_all_edges`

Reply:
I have named these properties as:
> map_reviewers_to_indexes 
> map_papers_to_indexes

Any suggestions for better var names is welcome.